src/main/java/org/fundacionjala/coding/jimmy/Kata2.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.coding.jimmy;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+/**
+ * This kata was performed by Jimmy Romero.
+ */
+public final class Kata2 {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)